config/locales/en.yml

```diff
@@ -520,7 +520,6 @@ en:
     admin_name: 'Administrator'
     heading:
       about: 'About'
-      help: 'Documentation & Support'
```

Comment:
@srerickson - Removing this translation value causes the app to raise `translation missing: en.navbar.heading.help` when we visit `/help`. 

Is the intent to keep the help page so that any external links to it still work, and to just change the `Help` link in the header to nav to `https://docs.scholarsphere.psu.edu/`? If we want to keep the Help page, we will need to re-add this.

Reply:
I'd almost suggest, in `routes.rb` just redirecting to docs.scholarsphere.psu.edu on gets to `/help`?

Reply:
Thanks - I like the redirect idea. Then I can delete `help.md` as well